src/Model/TestCollection.php

```diff
@@ -42,22 +42,22 @@ public function __construct(ConfigurationHolder $configurationHolder)
         $this->iteratorIterator = new \RecursiveIteratorIterator($this->iterator);
     }
 
-    public function current()
+    public function current(): mixed
```

Comment:
This library supports PHP >= 7.0 (according to the `composer.json`) while the `mixed` type is PHP >= 8.0 only.

You can use the [`#[ReturnTypeWillChange]`](https://php.watch/versions/8.1/ReturnTypeWillChange) attribute to fix the deprecation instead.

Reply:
@jdreesen Thanks so much! I should've checked that--please accept my apologies and thanks for the quick response! Fixing now.